source/views/settings/screens/overview/developer.tsx

```diff
@@ -1,25 +1,40 @@
 import * as Sentry from '@sentry/react-native'
 import * as React from 'react'
-import {Alert} from 'react-native'
-import {Section, PushButtonCell} from '@frogpond/tableview'
+import {Alert, StyleSheet} from 'react-native'
+import {
+	Section,
+	PushButtonCell,
+	CellTextField,
+	ButtonCell,
+} from '@frogpond/tableview'
 import type {NavigationScreenProp} from 'react-navigation'
 import {isDevMode} from '@frogpond/constants'
+import restart from 'react-native-restart'
+import * as storage from '../../../../lib/storage'
+import {PRODUCTION_SERVER_URL} from '../../../../lib/constants'
 
-type Props = {navigation: NavigationScreenProp<any>}
+type Props = {
+	navigation: NavigationScreenProp<any>
+}
+
+export const DeveloperSection = ({navigation}: Props): React.ReactElement => {
+	const [serverAddress, setServerAddress] = React.useState('')
+	const _serverAddress = React.useRef<CellTextField>(null)
```

Comment:
Would you mind calling this `serverAddressRef`? I use underscore-prefixed words to mean "variables that must exist but which I don't really care about the contents of".